components/_patterns/02-molecules/card/01-card.twig

```diff
@@ -1,51 +1,126 @@
+ * - card_subtitle_base_class - base classname of the subtitle component. Defaults to "subtitle"
+ * - card_subtitle_modifiers - array of modifiers to add to the base classname of the subtitle
+ *
+ * - card_body - the content of the body component
+ * - card_body_base_class - base classname of the body component. Defaults to "body"
+ * - card_body_modifiers - array of modifiers to add to the base classname of the body
+ *
+ * - card_link_text - the content of the link component
+ * - card_link_url - the url the link should link to
+ * - card_link_base_class - base classname of the link component. Defaults to "link"
+ * - card_link_attributes - array of attribute,value pairs for the link attribute
+ * - card_link_modifiers - array of modifiers to add to the base classname of the link
+ *
+ * - card_button_content - the content of the button component
+ * - card_button_url - the url the button should link to
+ * - card_button_base_class - base classname of the button component. Defaults to "button"
+ * - card_button_attributes - array of attribute,value pairs for the button attribute
+ * - card_button_modifiers - array of modifiers to add to the base classname of the button
+ */
+#}
+<div class="{{ card_base_class|default('card') }}{% for modifier in card_modifiers %} {{ card_base_class|default('card') }}--{{ modifier }}{% endfor %}{% if card_blockname %} {{ card_blockname }}__{{ card_base_class|default('card') }}{% endif %}">
+  {% if card_label %}
+    <h2 class="{{ card_label_base_class|default('card-label') }}{% for modifier in card_modifiers %} {{ card_label_base_class|default('card-label') }}--{{ modifier }}{% endfor %}{% if card_blockname %} {{ card_blockname }}__{{ card_label_base_class|default('card-label') }}{% endif %}">
+      {{ card_label }}
+    </h2>
   {% endif %}
-  {{ title_prefix }}
-  {% include "@atoms/02-text/00-headings/heading-2/heading-2.twig"
-    with {
-      "heading_2": card_title,
-      "heading_2_url": card_title_link,
-      "heading_2_blockname": card_title_blockname,
-      "heading_2_modifiers": card_title_modifiers
-    }
-  %}
-  {{ title_suffix }}
-  {% if card_sub %}
-    <div class="card__sub">
-      {% include "@atoms/02-text/text/01-paragraph.twig"
+  <div class="{{ card_inner_base_class|default('card-inner') }}{% for modifier in card_modifiers %} {{ card_inner_base_class|default('card-inner') }}--{{ modifier }}{% endfor %}{% if card_blockname %} {{ card_blockname }}__{{ card_inner_base_class|default('card-inner') }}{% endif %}">
+    {% if card_img_src %}
+      {% include "@atoms/04-images/00-image/image.twig"
         with {
-          "paragraph_content": card_sub,
-          "paragraph_modifiers": card_sub_modifiers
+          "img_src": card_img_src,
+          "img_alt": card_img_alt,
+          "image_base_class": card_img_base_class|default('image'),
+          "image_modifiers": card_img_modifiers,
+          "image_blockname": card_blockname|default('card'),
```

Comment:
We could also set a variable at the beginning of the file for this:
`{% set card_blockname_var = card_blockname|default('card') %}`

This is a micro performance optimization and might decrease readability ...